__tests__/APP.test.js

```diff
@@ -61,3 +61,59 @@ describe("task3- GET api/topics", () => {
       });
   });
 });
+
+describe.only("task5- GET api/articles/:articleiD", () => {
+  test("returns a single object (with correct properties article) when id exists", () => {
+    return request(app)
+      .get("/api/articles/3")
+      .expect(200)
+      .then(({ body }) => {
+        // console.log("in test", body.article);
+        const article = body.article;
+        expect(article).toMatchObject({
```

Comment:
Great to see you have used a flexible matcher here with toMatchObject. This allows for later extension of the endpoint. 👍